tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -200,18 +200,18 @@
 
     <!-- Integration: MongoDB.Driver.Core -->
     <!--    Assembly: MongoDB.Driver.Core -->
-    <!-- Latest package https://www.nuget.org/packages/MongoDB.Driver/2.28.0 -->
-    <PackageReference Include="MongoDB.Driver" Version="2.28.0" />
+    <!-- Latest package https://www.nuget.org/packages/MongoDB.Driver/2.29.0 -->
+    <PackageReference Include="MongoDB.Driver" Version="2.29.0" />
 
     <!-- Integration: MongoDB.Driver.Core -->
     <!--    Assembly: MongoDB.Driver.Core -->
-    <!-- Latest package https://www.nuget.org/packages/MongoDB.Driver.Core/2.28.0 -->
-    <PackageReference Include="MongoDB.Driver.Core" Version="2.28.0" />
+    <!-- Latest package https://www.nuget.org/packages/MongoDB.Driver.Core/2.29.0 -->
+    <PackageReference Include="MongoDB.Driver.Core" Version="2.29.0" />
 
     <!-- Integration: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
     <!--    Assembly: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/MSTest.TestAdapter/3.5.2 -->
-    <PackageReference Include="MSTest.TestAdapter" Version="3.5.2" />
```

Comment:
This one causes test failures @tonyredondo 😢 https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=164428&view=logs&j=64386644-9e17-59ba-c5e1-616d1f2330c1

Reply:
Excluded it from this PR for you to look into separately